src/app/receptionist/shift.cpp

```diff
@@ -0,0 +1,12 @@
+#include "shift.hpp"
+
+namespace
+{
+static std::map<Shift, std::string> shift_text{{Shift::Morning, "morning shift"},
+                                               {Shift::Afternoon, "afternoon shift"}};
+}
+
+std::string convertShiftToString(const Shift &shift)
+{
+    return shift_text.at(shift);
```

Comment:
Contrary to maps containing many values (especially with potential to add many of them), here a map might be a bit of an overkill.
simple if statements or switch (potentially with default throw() branch) might be better.

Reply:
I agree, yet I wanted to test it out after you mentioned in last time. 
I'll change that